src/string_input_popup.cpp

```diff
@@ -347,6 +355,20 @@ const std::string &string_input_popup::query_string( const bool loop, const bool
             ret.insert( _position, t );
             _position += t.length();
             redraw = true;
+            edit.erase(0);
+            ctxt->set_edittext(edit.c_str());
+        }
+        else if (ev.edit_refresh) {
```

Comment:
Styling: `} else if( x ) {` - closing braces, `else` and `if` should be on one line.